core/pom.xml

```diff
@@ -391,16 +391,42 @@
      placed in the "provided" scope, rather than the "compile" scope, and NoClassDefFoundError exceptions are handled
      when the user has not explicitly compiled with the Hive module.
     -->
+    <dependency>
```

Comment:
Do these dependencies have to come into core?

Reply:
I think this is okay because it uses the "core" version of hive-exec. The version without a classifier shades everything, so I think the intent here is to manage Hive's transitive dependencies in Spark instead of in a fork of Hive.

Reply:
Oh right it's `provided` anyway